src/main/java/frc/robot/OperatorBoard.java

```diff
@@ -11,15 +11,14 @@
 public class OperatorBoard {
 
     private static Joystick HARDWARE;
-    public OperatorBoardButton BUTTON_ONE;
-    public OperatorBoardButton BUTTON_TWO;
-    public OperatorBoardButton BUTTON_THREE;
-    public OperatorBoardButton BUTTON_SIX;
-    public OperatorBoardButton BUTTON_ELEVEN;
+    public OperatorBoardButton startShooter;
+    public OperatorBoardButton feeder;
+    public OperatorBoardButton intakeForward;
+    public OperatorBoardButton intakeReverse;
+    public OperatorBoardButton releaseRatchet;
+    public OperatorBoardButton releaseClimber;
+    public OperatorBoardButton resetClimber;
     public OperatorBoardButton BUTTON_TWELVE;
```

Comment:
Delaynie says change this to private because the button does not have a function